docs/content/users/usage/architecture.md

```diff
@@ -102,6 +102,27 @@ Files beginning with `.` are hidden because they shouldn’t be fiddled with; mo
 
 There’s only one global `.ddev` directory, which lives in your home directory: `~/.ddev` (`$HOME/.ddev`).
 
+!!!tip "Where is my global `.ddev` config?"
+    Use `ddev version` (look at `global-ddev-dir`) to check which location is currently used for the `.ddev` global directory.
+
+!!!tip "What if I don't want to clutter up my `$HOME` with a `.ddev` directory?"
+    DDEV can use the `$XDG_CONFIG_HOME` environment variable from [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) to move `~/.ddev` to the `$XDG_CONFIG_HOME/ddev` directory if `$XDG_CONFIG_HOME` is [defined](https://superuser.com/questions/365847/where-should-the-xdg-config-home-variable-be-defined/):
+
+    ```bash
+    # permanently set environment variable depending on your OS:
+    export XDG_CONFIG_HOME="/my-desired-path"
+    # restart the terminal and run:
+    mv ~/.ddev ${XDG_CONFIG_HOME}/ddev
+    ```
+
+    Otherwise (Linux and WSL2 only):
```

Comment:
> Otherwise, on Linux/WSL2 only, the default `$HOME/.config/ddev` will be used if `$HOME/.ddev` does not exist:

Not exactly, `$HOME/.config/ddev` will be used only if this directory exists, it doesn't depend on `$HOME/.ddev` existence.

Reply:
I think Randy has it right. The logic I'd expect, and that's specified by the XDG standard, is that `$HOME/.ddev` is used if and only if it already exists or `XDG_CONFIG_HOME=$HOME`.

This PR looks like it's doing the opposite: using `$HOME/.ddev` by default unless XDG_CONFIG_HOME is set or `$HOME/.config/ddev` already exists.